src/Microsoft.Identity.Client/Azure/Constants.cs

```diff
@@ -0,0 +1,173 @@
+using System;
+
+namespace Microsoft.Identity.Client.Azure
+{
+    /// <summary>
+    ///     Constants for environment variables, application settings and credential lookups
+    /// </summary>
+    public static class Constants
+    {
+        /// <summary>
+        ///     AzureTenantIdEnvName is the name of the environment variable which holds a string representation for a GUID,
+        ///     which is the ID of the tenant where the account resides
+        /// </summary>
+        public const string AzureTenantIdEnvName = "AZURE_TENANT_ID";
```

Comment:
`const` values in this file may do well to be converted to `static readonly` or get-only properties as they appear to be intended for consumption outside of the assembly where changes to the strings in future builds of this assembly should be reflected in its consumers

Reply:
Or see my remark below on the `Env` class about the .NET Core / Standard configuration mechanism

Reply:
Changed these definitions to be static readonly.

As per using .NET Core / Standard configuration, I would love to move to this, but I think this is a bit orthogonal. The Env class just provides the default mapping of "well known" env var variable names. 

Since the MSAL project depends on netstandard1.3 and [Microsoft.Extensions.Configuration](https://www.nuget.org/packages/Microsoft.Extensions.Configuration/) depends on netstandard2.0, how would you recommend I enable a consumer? Would it simply be by making a POCO with the standard names as the properties?